## HOJA DE RESPUESTA TRABAJO PRACTICO 

In [4]:
import pandas as pd
import numpy as np

MG = pd.read_excel("16_MATRICULAS_ED_SUPERIOR_MAGALLANES_2021.xlsx")

### ITEM 1

#### 1. Estudien la correlación entre el valor de la matrícula y el valor del arancel para toda la base de datos y luego separen por tipo de institución. ¿Los grupos se comportan parecido? ¿Vale la pena estudiar todos los grupos? Visualicen apropiadamente.

#### 2. ¿Cuál es la relación entre el arancel y la duración de la carrera correspondiente? ¿Es verdad que, mientras más larga la carrera, más cara? ¿Será igual en todas las áreas del conocimiento?

### ITEM 2

#### 1. ¿Se produjo una gran vuelta a la presencialidad o los formatos que incluyen la no presencialidad han ganado terreno?

In [6]:
modality_counts = MG['MODALIDAD'].value_counts()
modality_percentages = MG['MODALIDAD'].value_counts(normalize=True) * 100

# Mostrar los resultados
print("Cuenta de modalidades:")
print(modality_counts)
print("\nPorcentaje de modalidades:")
print(modality_percentages)

Cuenta de modalidades:
MODALIDAD
Presencial        5794
Semipresencial       2
Name: count, dtype: int64

Porcentaje de modalidades:
MODALIDAD
Presencial        99.965493
Semipresencial     0.034507
Name: proportion, dtype: float64


Se puede concluir que sí se produjo una gran vuelta a la presencialidad (o la presencialidad se mantuvo como el formato dominante antes de la pandemia), ya que el 99.97% de las matrículas correspondieron a la modalidad Presencial.

Los formatos que incluyen la no presencialidad (Semipresencial y A Distancia) tienen una participación prácticamente nula en el total de matrículas de la región para ese año.

#### 2. ¿Qué tipo de institución emplea de mayor manera formatos no exclusivamente presenciales?

#### 3. ¿Cuál es la probabilidad de seleccionar de los estudiantes hombres de la región a uno del área de educación? ¿Cuál es la probabilidad de seleccionar de las estudiantes mujeres de la región a una del área de educación? ¿Hay una discrepancia aparente? ¿Cómo se explica?

In [21]:
# Calculando Prob Hombre del área de educación
MG_Genero = MG[MG['GENERO'] == "Masculino"]
prob_areas = MG_Genero['AREA CONOCIMIENTO'].value_counts(normalize=True)
prob_final = prob_areas.get('Educacion')

print(f"Probabilidad de ser del área de educación (dado que es Hombre): {prob_final:.2%}")

# Calculando Prob Mujer del área de educación
MG_Genero = MG[MG['GENERO'] == "Femenino"]
prob_areas = MG_Genero['AREA CONOCIMIENTO'].value_counts(normalize=True)
prob_final = prob_areas.get('Educacion')

print(f"Probabilidad de ser del área de educación (dado que es Mujer): {prob_final:.2%}")

print("""
Hay una gran discrepancia entre hombres y mujeres, esto debido a la diferencia porcentual entre ambos géneros. 
Siendo las mujeres las que lideran esta área de conocimiento.
""")

Probabilidad de ser del área de educación (dado que es Hombre): 6.36%
Probabilidad de ser del área de educación (dado que es Mujer): 15.07%

Hay una gran discrepancia entre hombres y mujeres, esto debido a la diferencia porcentual entre ambos géneros. 
Siendo las mujeres las que lideran esta área de conocimiento.

